### PR TITLE
Add known issue to handle stuck prebackup when upgrading

### PR DESCRIPTION
**Pull request by ccremer**
## Summary

* Adds a note how to handle stuck prebackup pods after an upgrade

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`,
      as they show up in the changelog
- [x] Update the documentation.
- [x] ~Update tests.~
- [x] ~Link this PR to related issues.~

### CODE REVIEW

**Comment by Kidswiss** [comment]
Does this only happen directly after the upgrade?

**Comment by ccremer** [comment]
Hard to say. In the test cluster I have now also stuck backups longer after the upgrade:
```console
k -n syn-backup-cluster-objects get backups 
NAME                       SCHEDULE REF   COMPLETION   PREBACKUP   AGE
backupjob-1611590400       k8s-objects                             21h
backupjob-1611648000       k8s-objects    Succeeded    Succeeded   5h3m
backupjob-1611648900       k8s-objects    Succeeded    Succeeded   4h48m
backupjob-1611649800       k8s-objects    Succeeded    Succeeded   4h33m
k8s-objects-backup-2b58h   k8s-objects                 Waiting     93m
k8s-objects-backup-5lf45   k8s-objects                 Waiting     108m
k8s-objects-backup-67z6t   k8s-objects                 Waiting     78m
k8s-objects-backup-6jkvh   k8s-objects                 Waiting     63m
k8s-objects-backup-7646r   k8s-objects                 Waiting     3m43s
k8s-objects-backup-8mdt7   k8s-objects    Succeeded    Succeeded   4h3m
k8s-objects-backup-bh9d9   k8s-objects                 Waiting     3h18m
k8s-objects-backup-bqnrh   k8s-objects                 Waiting     33m
k8s-objects-backup-cjvmr   k8s-objects                 Waiting     18m
k8s-objects-backup-dz7rz   k8s-objects    Succeeded    Succeeded   4h18m
k8s-objects-backup-jqdqx   k8s-objects                 Waiting     123m
k8s-objects-backup-lk6sw   k8s-objects                 Waiting     168m
k8s-objects-backup-rhk8j   k8s-objects                 Waiting     138m
k8s-objects-backup-rns5w   k8s-objects                 Waiting     3h33m
k8s-objects-backup-wb49m   k8s-objects                 Waiting     48m
k8s-objects-backup-zhnv9   k8s-objects                 Succeeded   3h48m
k8s-objects-backup-zkg95   k8s-objects                 Waiting     3h3m
k8s-objects-backup-zp6wr   k8s-objects                 Waiting     153m
```
May be also related to #331

File: docs/modules/ROOT/pages/how-tos/upgrade.adoc [hunk 1]
```diff
@@ -61,3 +61,12 @@ kubectl -n ${ns} logs -l "app.kubernetes.io/name=k8up"
 # Trigger a new backup by creating a new Backup object
 kubectl create -f <your-backup-file-spec>
 ----
+
+=== Known issues
+
+- PreBackups get stuck
++
+If you are using PreBackupPods and multiple `Backups` are waiting for one to complete, they may get stuck waiting for PreBackup.
+In such a case, you need to delete the oldest waiting `Backup` jobs to keep 1 object only.
```
Comment (cimnine) [inline docs/modules/ROOT/pages/how-tos/upgrade.adoc:70]:
```suggestion
In such a case, you need to just keep the most recent `Backup` job and delete all the older ones.
```

Also: Backup _jobs_ or Backup _resources_?

File: docs/modules/ROOT/pages/how-tos/upgrade.adoc [hunk 2]
```diff
@@ -61,3 +61,12 @@ kubectl -n ${ns} logs -l "app.kubernetes.io/name=k8up"
...
+Alternatively, you can delete all `Backup` jobs and just wait for the next schedule.
+That should make it easier to script if you encounter this problem in a large scale.
```
Comment (cimnine) [inline docs/modules/ROOT/pages/how-tos/upgrade.adoc:72]:
```suggestion
That is easier to script should you encounter this problem on a large scale.
```

**Review by cimnine — APPROVED** [review]

**Comment by ccremer** [comment]
I would suggest to wait for merging until we know more about #331

**Comment by Kidswiss** [comment]
I believe this isn't relevant anymore?

**Pull request closed without merging**